ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -278,9 +278,15 @@ describe('DataAccessRequirements', () => {
         expect(findCompletionBanner(wrapper).exists()).toBeTruthy();
     });
 
+    const repeat = async(n, callback) => await Promise.all(Array(n).fill(0).map(callback));
+    const TEST_ITERATIONS = 1;  // 1 passes, 2 fails
+    let currentIteration = 0;
+
     // RAS launch bug (no JIRA ticket)
     // temporarily disabled for flakiness - https://precisionmedicineinitiative.atlassian.net/browse/RW-7364
-    xit('should render all modules as complete by transitioning to all complete', async() => {
+    it('should render all modules as complete by transitioning to all complete', async() => repeat(TEST_ITERATIONS, async() => {
```

Comment:
When I run this locally, I see many errors about updates being applied while unmounted. There seem to be a lot of issues with writing this kind of dynamically updating test case which involve direct store interactions.

One way of dealing with this is to use fake jest timers. An example of that is here: runtime-utils.spec.tsx . It's possible this resolves the issue, but again - I haven't been able to repro so it's hard to say.

Reply:
OK, let's give it a try at #5699